Switch event and venue details content order when using small screen

nikomakela opened the pull request:
LIIKUNTA-759.

Take advantage of grid styles to reorder content when a small screen breakpoint is window size reached.

When using bigger than mobile sized screen, the content order would be:
1. event description
2. share buttons,
3. map,
4. sidebar
5. other events, footer, etc...

When using mobile sized screen, switch the sidebar with secondary data (location / map, share buttons...) in main content, so in mobile the order would be:
1. event description
2. sidebar
3. share buttons,
4. map,
5. other events, footer, etc...

This commit also contains a small trivial style change: Use aside element for event details sidebar.


HCRC-lib pull request about same matter: https://github.com/City-of-Helsinki/react-helsinki-headless-cms/pull/212

nikomakela committed:
feat: switch event details content order when using small screen

LIIKUNTA-759.

Take advantage of grid styles to reorder content when a small screen
breakpoint is window size reached.

When using bigger than mobile sized screen, the content order would be:
1. event description
2. share buttons,
3. map,
4. sidebar
5. other events, footer, etc...

When using mobile sized screen, switch the sidebar with secondary data
(location / map, share buttons...) in main content, so in mobile the
order would be:
1. event description
2. sidebar
3. share buttons,
4. map,
5. other events, footer, etc...

This commit also contains a small trivial style change: Use aside
element for event details sidebar

nikomakela committed:
feat: switch venue details content order when using small screen

LIIKUNTA-759.

Take advantage of grid styles to reorder content when a small screen
breakpoint is window size reached.

When using bigger than mobile sized screen, the content order would be:
1. venue description
2. share buttons,
3. map,
4. sidebar
5. venue events, footer, etc...

When using mobile sized screen, switch the sidebar with secondary data
(location / map, share buttons...) in main content, so in mobile the
order would be:
1. venue description
2. sidebar
3. share buttons,
4. map,
5. venue events, footer, etc...

terovirtanen commented:
**FEDERATION-ROUTER branch is deployed to platta: https://events-graphql-federation-events-pr885.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**LIIKUNTA-UI branch is deployed to platta: https://liikunta-pr885.dev.hel.ninja** :rocket::rocket::rocket:

nikomakela committed:
feat: switch page content order when using small screen

Upgrade HCRC-lib to get new grid style changes on article and dynamic
CMS pages. Also fix event details remainingAttendeeCapacity field.

LIIKUNTA-759.

Take advantage of grid styles to reorder content when a small screen
breakpoint is window size reached.

When using bigger than mobile sized screen, the content order would be:
1. page main (text) content
2. share buttons,
3. sidebar
4. modules, footer, etc...

When using mobile sized screen, switch the sidebar with secondary data
(share buttons...) in main content, so in mobile the
order would be:
1. page main (text) content
2. sidebar
3. share buttons,
5. modules, footer, etc...

terovirtanen commented:
**FEDERATION-ROUTER branch is deployed to platta: https://events-graphql-federation-hobbies-pr885.dev.hel.ninja** :rocket::rocket::rocket:
